files/server/pfdel

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/perl -w
+#
+# pfdel - Deletes message containing specified address from the Postfix queue.
+#         Matches either sender or recipient address.
+#
+# Usage: pfdel <email_address>
+#
+
+use strict;
+
+# Change these paths if necessary.
+my $LISTQ = "/usr/sbin/postqueue -p";
+my $POSTSUPER = "/usr/sbin/postsuper";
+
+my $email_addr = "";
+my $qid = "";
+my $euid = $>;
+
+if ( @ARGV !=  1 ) {
+	die "Usage: pfdel <email_address>\n";
+} else {
+	$email_addr = $ARGV[0];
```

Comment:
Can you please convert all tabs to spaces and also convert this to two space indents like we do with ruby code?